src/include/processor/operator/persistent/node_batch_insert.h

```diff
@@ -135,6 +133,8 @@ class NodeBatchInsert final : public BatchInsert {
         std::optional<IndexBuilder>& indexBuilder, storage::MemoryManager* mm) const;
 
 private:
+    std::string tableName;
```

Comment:
It's fine to keep it here. After u change RelBatchInsert, this can be moved to base class.